modules/apps/portal-cache/portal-cache-impl/src/main/java/com/liferay/portal/cache/internal/dao/orm/EntityCacheImpl.java

```diff
@@ -291,6 +283,24 @@ protected void activate() {
 			portalCacheManager = _multiVMPool.getPortalCacheManager();
 
 		portalCacheManager.registerPortalCacheManagerListener(this);
+
+		_serviceRegistration = bundleContext.registerService(
+			CacheRegistryItem.class, new EntityCacheCacheRegistryItem(), null);
+	}
+
+	@Deactivate
+	protected void deactivate() {
+		_serviceRegistration.unregister();
+	}
+
+	private void _cleanCache() {
+		_notifyFinderCache(null, null, false);
```

Comment:
Hey @GabrielHenriquedev, you do not need to create this private method. You can call the clearCache directly into the EntityCacheCacheRegistryItem class.